### PR TITLE
Removes the simulator and related code

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

File: docs/process.rst [hunk 1]
```diff
@@ -219,8 +219,7 @@ Utopic) will run into problems with running the functional tests.
   string that combines the value in the branch's ``CHANGES.yaml`` with the
```
Comment (jaycee) [inline docs/process.rst]: This file is massively out of date; I corrected the bits relevant to this branch, but someone should take a look at this whole file.

**Comment by frankban** [comment]
👍 thank you for removing all this obsolete unsupported code!

**Comment by jaycee** [comment]
QA:

* Spin up the GUI; bring up help ('?' brings up help). Ensure that the simulator isn't mentioned there.
* Hit 'Ctrl+s'; the simulator should not start, and there should be no errors. You may get a save dialog, depending on os/browser.
* Sandbox should otherwise work normally.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5306/
Test PASSed.

**Comment by Lucapaulina** [comment]
I love the simulator....goodbye old friend.

**Comment by Lucapaulina** [comment]
We use the simulator quite extensively to generate screenshots, is this
something that absolutely needs to be removed?

**Comment by bac** [comment]
QA is iffy.  Look at this screenshot of the help screen (Chrome on OS X)

screen shot 2016-06-27 at 9 11 23

Lots of stuff on the RHS is a mess. I compared to demo.jujucharms.com and it is not similarly mixed up.

**Comment by bac** [comment]
Since the old layout is pretty messed up on several browsers, go ahead and land this as is* but create a card for cleaning up the help screen layout.

Thanks.

*modulo @Lucapaulina's concerns.

**Comment by jaycee** [comment]
@Lucapaulina So it's fairly simple using the browser's js console to put any arbitrary unit into an error state. If we provided the directions to you of how to do that (including the commands that you could just copy paste) would that be an adequate replacement?

It would have the added benefit of you not having to wait for a unit to eventually trigger an error.

**Comment by Lucapaulina** [comment]
@jaycee sure, that would work for us :)

**Comment by jaycee** [comment]
I've created directions for how to create error states in the GUI to @Lucapaulina and made a card regarding the help menu issues for @bac.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**